starters/basic-starter/pages/[[...slug]].tsx

```diff
@@ -49,7 +48,9 @@ export default function NodePage({ node, menus }: NodePageProps) {
 // See https://nextjs.org/docs/basic-features/data-fetching/get-static-paths.
 export async function getStaticPaths(context): Promise<GetStaticPathsResult> {
   return {
-    paths: await getPathsFromContext(CONTENT_TYPES, context),
```

Comment:
I think we should create wiki page that documents how this could be done since this still makes sense for smaller websites